DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factory/NewDicomProcessorFactory.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors.Factory
+{
+    public class NewDicomProcessorFactory
```

Comment:
>NewDicomProcessorFactory [](http://example.com/codeflow?start=17&length=24)

I think you can ignore this for now. You simply create a class and never use it or create some logics to process it, so why not remove it for now and add it back when you need to do that?

Reply:
Removed it, thanks!